common/src/main/java/com/wynntils/features/user/map/MapFeature.java

```diff
@@ -131,6 +136,15 @@ private void openMainMap() {
         McUtils.mc().setScreen(MainMapScreen.create());
     }
 
+    private void openWaypointSetup() {
+        var location = new PoiLocation(
```

Comment:
```suggestion
        PoiLocation location = new PoiLocation(
```

We do not use var anywhere in Artemis. And we won't.